### ISSUE TITLE
Support for curseforge slugs

### ISSUE THREAD

**Issue by KyleUltimate** [theRookieCoder/ferium#68]
_Originally posted by @theRookieCoder in https://github.com/theRookieCoder/ferium/issues/65#issuecomment-1125065106_
Detailed discussion in https://github.com/theRookieCoder/ferium/issues/65

**Comment by theRookieCoder** [comment]
This will be more convenient as slugs are in the URL

**Comment by KyleUltimate** [comment]
Yep, that's quite useful because now you can turn a list of curseforge urls and regex the slugs out.
After that using GNU `parallel` to add the mods to ferium multi thread!
This can open up for a lot of new potentials!
